static/app/views/relocation/relocation.spec.tsx

```diff
@@ -54,31 +55,9 @@ describe('Relocation', function () {
       renderPage('encrypt-backup');
       expect(
         await screen.findByText(
-          'Create an encrypted back up of current self-hosted instance'
-        )
-      ).toBeInTheDocument();
-      expect(
-        await screen.findByText(
-          'You’ll need to have the public key saved in the previous step accessible and run the command below in your terminal to ensure success'
-        )
-      ).toBeInTheDocument();
-      expect(await screen.findByText('Understanding the command:')).toBeInTheDocument();
-      expect(await screen.findByText('./sentry-admin.sh')).toBeInTheDocument();
```

Comment:
Let's keep this in, since it's not really copy, it's a specific command that we want shown on screen.

Reply:
sounds good